android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsDebugModule.java

```diff
@@ -0,0 +1,34 @@
+package io.invertase.googlemobileads;
+
+import android.os.Handler;
+import android.os.Looper;
+import android.util.Log;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactMethod;
+import com.google.android.gms.ads.MobileAds;
+import io.invertase.googlemobileads.common.ReactNativeModule;
+
+class ReactNativeGoogleMobileAdsDebugModule extends ReactNativeModule {
+  public static final String SERVICE = "RNGoogleMobileAdsDebugModule";
+
+  public ReactNativeGoogleMobileAdsDebugModule(ReactApplicationContext reactContext) {
+    super(reactContext, SERVICE);
+  }
+
+  @ReactMethod
+  public void openDebugMenu(final String adUnit) {
+    new Handler(Looper.getMainLooper())
+        .post(
+            new Runnable() {
+              @Override
+              public void run() {
+                try {
+                  MobileAds.initialize(getReactApplicationContext());
```

Comment:
Is this extra initialize needed? I think we can leave this line out and just add a note to the interface that initialize has to be called before using this method.